frontend/src/pages/ForgotPasswordRecovery/ForgotPasswordRecovery.jsx

```diff
@@ -0,0 +1,139 @@
+          ></img>
+        </div>
+
+        <div
+          className={`${style["forgot-password-recovery-form"]} ${style["child2"]}`}
+        >
+          <div className={style["forgot-password-recovery-card"]}>
+            {checkUpdatedPassword ? (
+              <React.Fragment>
+                <h1 className={style["card-heading"]}>Password Updated</h1>
+                <div className={style["inside-card"]}>
+                  <p style={{ textAlign: "center" }}>
+                    Your Password has been updated. Please login with your
+                    updated password
+                  </p>
+                  <Grid container spacing={3}>
+                    <Grid xs={12} style={{ padding: "5%" }}>
+                      <Link to="/admin">
+                        <div className={style["submit-btn"]}>
+                          <Button2
+                            id="btn"
+                            label="Submit"
+                            type="submit"
+                            className={style["submit-btn-text"]}
+                          />
+                        </div>
+                        {console.log(style["back-button"])}
+                      </Link>
+                    </Grid>
+                  </Grid>
+                  <br />
+                </div>
+              </React.Fragment>
+            ) : (
+              <React.Fragment>
+                <h1 className={style["card-heading"]}>Reset your Password</h1>
+                <form onSubmit={handleSubmit}>
+                  <div className={style["inside-card"]}>
+                    <div className={style["forgot-password-recovery-input"]}>
+                      <input
+                        type="password"
+                        id="new password"
+                        required="required"
+                        name="new password"
+                        placeholder="New Password"
+                        value={new_password}
+                        className={style["input-forgot-password-recovery"]}
+                        onChange={(e) => {
+                          setNewPassword(e.target.value);
+                        }}
+                      ></input>
+
+                      <input
+                        type="password"
+                        id="confirm password"
+                        name="confirm password"
+                        required="required"
+                        value={confirm_password}
+                        placeholder="Confirm Password"
+                        className={style["input-forgot-password-recovery"]}
+                        onChange={(e) => {
+                          setConfirmPassword(e.target.value);
+                        }}
+                      ></input>
+                    </div>
+
+                    <div className={style["submit-btn"]}>
+                      <Button2
+                        type="submit"
+                        label="Submit"
```

Comment:
Please label it as `Login` as we will be directing the user to Login Form after they click on it

Reply:
I have completed all the changes and for the "forgot password" route I was checking my code on that route and I forgot to update it. Sorry about that. 
Now I have updated all of the things and removed the conflicts too. 
Let me know if there is an issue. Also, the image is there in this"public/images" folder. I am able to see it. Kindly tell me if you are able to see it or not?